cogs/login.py

```diff
@@ -0,0 +1,146 @@
+        )
+        embed.add_field(
+            name = ":nazar_amulet: Viki (KDramas) :nazar_amulet:",
+            value = "u: " + config["Viki"]["login"] + "\np: " + config["Viki"]["password"], 
+            inline=True
+        )
+        embed.add_field(
+            name = ":brown_circle: Showtime :brown_circle:",
+            value = "u: " + config["Showtime"]["login"] + "\np: " + config["Showtime"]["password"], 
+            inline=True
+        )
+
+        select = Select(                                        # Select Method that Creates Selection Menu for Embed
+            placeholder= "Choose a streaming service...",       # Placeholder Text
+            options=[                                           # Selection Options
+                discord.SelectOption(
+                    label="Crunchyroll", 
+                    emoji="🟠", 
+                    description="Crunchyroll Login"),
+                discord.SelectOption(
+                    label="Disney+", 
+                    emoji="🔵", 
+                    description="Disney+ Login"),
+                discord.SelectOption(
+                    label="HBO Max", 
+                    emoji="⚫", 
+                    description="HBO Max Login"),
+                discord.SelectOption(
+                    label="Hulu", 
+                    emoji="🟢", 
+                    description="Hulu Login"),          
+                discord.SelectOption(
+                    label="Netflix", 
+                    emoji="🔴", 
+                    description="Netflix Login"),
+                discord.SelectOption(
+                    label="NFL+", 
+                    emoji="🏈", 
+                    description="NFL+ Login"),
+                discord.SelectOption(
+                    label="Paramount+", 
+                    emoji="🗻", 
+                    description="Paramount+ Login"),
+                discord.SelectOption(
+                    label="Peacock", 
+                    emoji="🦚", 
+                    description="Peacock Login"),
+                discord.SelectOption(
+                    label="Viki", 
+                    emoji="🧿", 
+                    description="Viki Login"),
+                discord.SelectOption(
+                    label="Showtime", 
+                    emoji="🟤", 
+                    description="Showtime Login")           
+            ]
+        )
+
+        async def my_callback(interaction):
+            if interaction.user == ctx.author:      # Interaction Author Must be Original Sender
+                await interaction.response.send_message(config[select.values[0]]["password"])
+
+        select.callback = my_callback
+        view = View(timeout=15)             # Disables after  15 Second
+        view.add_item(select)
+        await ctx.channel.send(embed=embed, view=view)
+    
+    @classmethod
+    def is_enabled(cls, configs: Dict[str, Any] = {}):
+        return (
+            configs["ENABLE_LOGIN"]
```

Comment:
Add a boolean to check if login.json is openable and exists

Reply:
idk how to do this